dkpro-core-io-lxf-asl/src/test/java/org/dkpro/core/io/lxf/LxfConverterTest.java

```diff
@@ -46,84 +46,85 @@ public void testText()
         throws Exception
     {
         Map<String, String> ids = new HashMap<>();
-        test("src/test/resources/lxf/delta/text/orig.lxf", false, ids);
+        test("src/test/resources/lxf/text/orig.lxf", false, ids);
```

Comment:
They are practically the same test files.  I wanted to use the same set for the tests.